playfab-docs/api-references/events/multiplayer-server-build-alias-created.md

```diff
@@ -0,0 +1,45 @@
+
+---
+title: multiplayer_server_build_alias_created 
+author: vorelien
+description: multiplayer_server_build_alias_created event.
+ms.author: vorelien
+ms.date: 02/19/2019
+ms.topic: article
+ms.service: azure-playfab
+keywords: playfab, playstream events
+ms.localizationpriority: medium
+---
+
+# multiplayer_server_build_alias_created 
+
+This event is triggered when a multiplayer server build alias is created.
+
+## Properties
+
+|Name|Type|Description|
+| :--------------------|:-------------------|:----------------------|
+|EntityLineage|[EntityLineage](data-types/entitylineage.md)|Entities that this entity is a child of.|
+|OriginalEventId|String|The original unique identifier associated with this event before it was posted to PlayFab. The value might differ from the EventId value, which is assigned when the event is received by the server.|
+|OriginalTimestamp|DateTime|The original time (in UTC) associated with this event before it was posted to PlayFab. The value might differ from the Timestamp value, which is set at the time the event is received by the server.|
+|Payload|[MultiplayerServerBuildAliasCreatedEventPaylod](data-types/multiplayerserverbuildaliascreatedeventpayload.md)|The multiplayer server build alias created event payload.|
+|WriterEntity|[EntityKey](data-types/entitykey.md)|Entity that wrote this event, included only if different than the event's entity.|
+
+## Common Properties
```

Comment:
I don't know if this is possible but is there any way to save these common properties as a template of some sort that all the pages use? Like, if we have to make a change to these it'd be nice to only make a change in 1 place